llama-index-integrations/tools/llama-index-tools-cassandra/llama_index/tools/cassandra/base.py

```diff
@@ -0,0 +1,84 @@
+"""Tools for interacting with an Apache Cassandra database."""
+from typing import List
+
+from llama_index.core.readers.base import BaseReader
+from llama_index.core.schema import Document
+from llama_index.core.tools.tool_spec.base import BaseToolSpec
+
+
+from pydantic import Field
+
+from llama_index.tools.cassandra.cassandra_database_wrapper import (
+    CassandraDatabase,
+)
+
+
+class CassandraDatabaseToolSpec(BaseToolSpec, BaseReader):
```

Comment:
Any reason to inherit base reader? None of the methods from base reader end up getting implemented here

Reply:
Apparently, not for any reason at all! I was looking at some of the other tools and saw they inherited BaseReader, so I made the (wrong) assumption that something about a BaseTool needed it. Mypy didn't say anything about unused Classes so I moved on. 

Removed and rebased.